internal/bminventory/inventory.go

```diff
@@ -2297,3 +2309,22 @@ func computeClusterProxyHash(httpProxy, httpsProxy, noProxy *string) (string, er
 	bs := h.Sum(nil)
 	return fmt.Sprintf("%x", bs), nil
 }
+
+func validateProxySettings(httpProxy, httpsProxy, noProxy *string) error {
+	if httpProxy != nil && *httpProxy != "" {
+		if err := validations.ValidateHTTPProxyFormat(*httpProxy); err != nil {
+			return common.NewApiError(http.StatusBadRequest, err)
+		}
+	}
+	if httpsProxy != nil && *httpsProxy != "" {
+		if err := validations.ValidateHTTPProxyFormat(*httpsProxy); err != nil {
```

Comment:
HTTP?, this will always fail no?

Reply:
no, the value for the httpsProxy should be a url with 'http' schema only.